pkg/brokerapi/service_instance.go

```diff
@@ -63,6 +63,13 @@ type CreateServiceInstanceResponse struct {
 	Operation    string `json:"operation, omitempty"`
 }
 
+// GetServiceInstanceResponse represents the response from a broker with a status
```

Comment:
Why are we adding this? There's already LastOperationResponse which is more appropriate than GetServiceInstance?